src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/LayerDescriptor.java

```diff
@@ -59,6 +65,8 @@ public String getUrlFieldName() {
         return urlField;
     }
 
+    public String getVariableName() { return variable; }
```

Comment:
I think variable is what we are talking about here - although NetCDFVariable may be clearer